internal/app/connectconformance/testsuites/grpc-web-client.yaml

```diff
@@ -0,0 +1,570 @@
+name: gRPC-web
```

Comment:
I created this file because running the client through the regular test files causes a few issues. The common thread for all of these is that there are some failures that are specific to the gRPC-web client, but _not_ with Connect via gRPC-web and we have no way to exclude the one but not the other.

* The biggest obstacle is cancellations with server-streaming. We can't really test cancellations because of the issues around `mode=grpcweb` and `mode=grpcwebtext` and that Connect doesn't support the latter (see docs [here](https://github.com/grpc/grpc-web?tab=readme-ov-file#streaming-support) and [here](https://connectrpc.com/docs/go/grpc-compatibility/#handlers)). Typically, we could just remove gRPC-web from the `relevantProtocols` list in the test file, but, we _can_ test cancellations with the Connect client using the gRPC-web protocol so that's a viable solution. We would need some way to say 'run these tests over gRPC-web using the Connect client only'.
* There is an issue with timeouts also (see [TODO](https://github.com/connectrpc/conformance/pull/751/files#diff-a8509409d477ca9786a1bcee8d03bed9c0e9e47501c85f657705510de01ff0a0R264) in this file). For some reason, timeouts of 200ms are flaky unless the server response delay is 1s or more. I'm not entirely sure if this is something on the gRPC-web side or how Connect is calculating the `Grpc-Timeout` header, but the same reason applies as cancellations -- we can test timeouts with a lower response delay using the Connect client via gRPC-web. We could increate the response delay in `timeouts.yaml` but we should probably figure out the actual issue first and just keep them isolated here (at least IMO).
* The `client_message_size.yaml` tests are hanging while trying to read the expanded requests from stdin (see TODO). For some reason, the `readReqBuffers` function in `conformance-runner.ts` is not reading these values properly. The consequence is the same as the above. This seems to only affect the gRPC-web client so we can't exclude the gRPC-web protocol from that file.
* Finally, we have the issue with trailers and trailer values with the same name ([TODO](https://github.com/connectrpc/conformance/pull/751/files#diff-a8509409d477ca9786a1bcee8d03bed9c0e9e47501c85f657705510de01ff0a0R28)). As discussed offline, we could mitigate that by simply creating a `duplicate_metadata.yaml`, but even then, we have the same problem of that working with Connect via gRPC-web vs. not working with the actual gRPC-web client.

Reply:
One way we could fix the issue of 'this works with Connect over gRPC-web but not with the gRPC-web client' is to use an expected failures file that contains the test we expect to fail with the gRPC-web client only.

Reply:
Will do. I added a TODO to this `grpc-web-client.yaml` file to fix this, which I'll do in a follow-up PR.